### ISSUE TITLE
[Feature Request] Option to lock view and aim together.

### ISSUE THREAD

**Issue by crz-cz** [KozGit/DOOM-3-BFG-VR#255]
I still using DK2 and x360 controller and only way I can control the aim now (hand with weapon) is with the right stick on x360 controller = unplayable. I dont have any kind of motion controllers (and dont plan to buy them in near future), so I cant use the motion controls for aiming.

The best way would be to have an option/cvar to get both hands (hand with flashlight and weapon hand or at least weapon hand) locked to my sight (HMD position tracking). The same way as it is in the old Doom3 BFG "VR" mods, where you always aim at the point where you look at that moment. Is it possible?

Im sorry if the option is already present, but I cant find out how to enable it.

**Comment by sycspysycspy** [comment]
It will be an unpleasant experience. if you happen to try g3doom you would know what I mean. You can hardly aim and the motion tickness will drive you mad.

**Comment by crz-cz** [comment]
I have to disagree. Its true that gz3doom can be very unpleasent, but from my experience its mostly because of the combination of movement and speed, aiming with head is perfectly ok and its not a problem (at least in my case).

Regarding the movement sickness so far this DOOM 3 BFG VR mod offers best solution that works and realy helps (atleast me) especialy thanks to "Slow Mo". I dont experience any sickness with "slow mo" (as long as Im standing and turn myself in real). I certainly agree that motion controls for hands help with immersion and may even reduce the motion sickness for some people. But thats not my case, so far I can say that motion controls does not affect my motion sickness, but movement speed does (a lot). I also have to disagree that one can hardly aim with aiming locked to view, on the contrary, while it may not be easy for everybody, I find it very easy and simple. 

The option to lock aiming and view togheter wont be for everybody, but certainly wont be causing unpleasent experience to all, but may help a lot to some. Thats why I think that option to lock aiming to view should certainly be present;)

**Comment by BiatuAutMiahn** [comment]
looking to aim would be preferable  in my case with ALVR/GearVR/Xbox Controller. the aim with the controller is a nightmare. I'd opt for the right stick for turning, and aiming to be in the direction of the gaze (with configurable offset)
